web/source/kmwuitoggle.js

```diff
@@ -64,13 +64,6 @@ if(!window['tavultesoft']['keymanweb']['ui']['name']) {
         someElement=window.event.srcElement;
       }
 
-      // Hides the Toggle interface if a control has KeymanWeb disabled by API calls.
-      if(activeControl){
-        if(!activeControl['LEnabled']) {
```

Comment:
This was conditioned on an old legacy property; it's no longer present, and a different object for the control is being passed in now that was causing the last line's test to trigger and fail when it shouldn't be tested at all.